Source/WebCore/platform/cf/RunLoopObserverCF.cpp

```diff
@@ -23,28 +23,56 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. 
  */
 
-#include "config.h"
-#include "RunLoopObserver.h"
+#import "config.h"
+#import "RunLoopObserver.h"
+
+#import <CoreFoundation/CoreFoundation.h>
 
 namespace WebCore {
 
-RunLoopObserver::~RunLoopObserver()
+static constexpr CFIndex cfRunLoopOrder(RunLoopObserver::WellKnownOrder order)
 {
-    invalidate();
+    constexpr auto coreAnimationCommit = 2000000;
```

Comment:
I'm guessing this is an invariant... is it defined anywhere?

Reply:
It's an intentionally chosen constant, that matches the runloop order in QuartzCore (it's hard-coded there, and not exposed as any symbol).